### PR TITLE
MAT-6393: Massage data if oid no url provided

### PR DESCRIPTION
**Pull request by mcmcphillips**
## Terminology Service PR

Jira Ticket: [MAT-6393](https://jira.cms.gov/browse/MAT-0000)
(Optional) Related Tickets:

### Summary
Minor fix

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -46,3 +46,9 @@ spring:
 mapping:
   data:
     code-system-entry-url: ${CODE_SYSTEM_ENTRY_URL:https://madie-dev-static.s3.amazonaws.com/mappings/code-system-entry.json}
+
+logging:
+  level:
+    root: INFO
+    gov.cms.madie.terminology: DEBUG
```
Comment (chubert-sb) [inline src/main/resources/application.yml]: We might want to put this one to info at least and just put it to debug as needed for special occasions

**Review by chubert-sb — APPROVED** [review]
